docker/install/ubuntu_install_zephyr.sh

```diff
@@ -58,10 +78,4 @@ chmod -R o+w ${ZEPHYR_PROJECT_PATH}
 mkdir zephyr/.cache
 chmod o+rwx zephyr/.cache
 
-pip3 install -r /opt/zephyrproject/zephyr/scripts/requirements.txt
-
-# the requirements above overwrite junintparser with an older version, but it is not
-# used so overwrite it again with the correct version
-pip3 install junitparser==2.4.2
```

Comment:
Isn't this just using the system Python with `python_cmd` here? This should still re-install junitparser since that was a hack to get a specific version working in CI which didn't agree with zephyr's requirements

Reply:
i think i might have removed pip3 to stop ppl from installing stuff outside the tvm venv